### ISSUE TITLE
Investigate two potential data issues from ClinVar

### ISSUE THREAD

**Issue by tskir** [opentargets/issues#3183]
Created from this community question: https://community.opentargets.org/t/genetic-disorder-of-clinvar-in-eva/1316

The user reports two issues:
* Very unspecific "genetic disorder" terms — this almost certainly comes from ClinVar and we can't do anything about it
* A missing association between SLC10A1 and hypercholanemia which is, according to the user, is reported in ClinVar but does not show up in the Platform

**Comment by tskir** [comment]
Investigated both questions and replied in the community thread. The first issue is indeed due to ClinVar having very nonspecific (essentially meaningless) terms; the second is due to EFO version sync — the term is very new (EFO v3.60.0) and so wasn't picked up by the ETL, which used EFO v3.58.0.

On that second point, I was wondering if we could review our release preparation processes. While the release was done in December, the EFO version we ended up using (v3.58.0) was released in September. A three month lag isn't terrible, but I think it would certainly be improved.

CC @d0choa @ireneisdoomed @DSuveges for information and potentially discussion.
